cli/dcoscli/node/main.py

```diff
@@ -23,7 +23,6 @@
     --version               Show version
 """
 
-import os
```

Comment:
> I like --option better because the "ssh" is implicit. This is an ssh command.

I thought the command was `dcos service log` in which case it is not obvious that `--option` refers to ssh options, no?